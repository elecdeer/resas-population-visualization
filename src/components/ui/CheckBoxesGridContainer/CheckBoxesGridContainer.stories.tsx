import {
  CheckBoxesGridContainer,
  CheckBoxesGridContainerProps,
} from "./CheckBoxesGridContainer";
import { CheckBox } from "../CheckBox/CheckBox";

export default {
  title: "ui/CheckBoxesGridContainer",
  component: CheckBoxesGridContainer,
};

export const Default = (args: CheckBoxesGridContainerProps) => (
  <CheckBoxesGridContainer {...args}>
    <CheckBox key={"box1"}>CheckBox1</CheckBox>
    <CheckBox key={"box2"}>CheckBox2</CheckBox>
    <CheckBox key={"box3"}>CheckBox3</CheckBox>
    <CheckBox key={"box4"}>CheckBox4</CheckBox>
    <CheckBox key={"box5"}>CheckBox5</CheckBox>
    <CheckBox key={"box6"}>CheckBox6</CheckBox>
    <CheckBox key={"box7"}>CheckBox7</CheckBox>
  </CheckBoxesGridContainer>
);
