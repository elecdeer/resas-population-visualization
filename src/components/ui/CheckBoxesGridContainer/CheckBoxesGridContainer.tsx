import React from "react";
import styles from "./CheckBoxesGridContainer.module.scss";

export type CheckBoxesGridContainerProps = {};

export const CheckBoxesGridContainer: React.FC<CheckBoxesGridContainerProps> =
  ({ children }) => {
    return (
      <div className={styles["check-boxes-grid-container"]}>{children}</div>
    );
  };
