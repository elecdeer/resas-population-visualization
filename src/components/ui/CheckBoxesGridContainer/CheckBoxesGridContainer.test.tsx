import { CheckBoxesGridContainer } from "./CheckBoxesGridContainer";
import { render } from "@testing-library/react";

test("CheckBoxesGridContainer Component Snapshot Test", () => {
  const { asFragment } = render(<CheckBoxesGridContainer />);
  expect(asFragment()).toMatchSnapshot();
});