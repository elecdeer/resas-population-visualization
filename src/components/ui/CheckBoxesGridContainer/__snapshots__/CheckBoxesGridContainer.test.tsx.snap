// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBoxesGridContainer Component Snapshot Test 1`] = `
<DocumentFragment>
  <div
    class="check-boxes-grid-container"
  />
</DocumentFragment>
`;
